edb/pgsql/metaschema.py

```diff
@@ -2094,39 +2081,22 @@ class StringSliceImplFunction(dbops.Function):
     """Get a string slice."""
     text = r'''
         SELECT
-            CASE
-                WHEN start IS NULL THEN
-                    edgedb._substr(
-                        val,
-                        1,
-                        edgedb._normalize_array_index(
-                            stop, edgedb._length(val)) - 1
-                    )
-                WHEN stop IS NULL THEN
-                    substr(
-                        val,
-                        edgedb._normalize_array_index(
-                            start, edgedb._length(val))
-                    )
-                ELSE
-                    edgedb._substr(
-                        val,
-                        edgedb._normalize_array_index(
-                            start, edgedb._length(val)),
-                        edgedb._normalize_array_index(
-                            stop, edgedb._length(val)) -
-                        edgedb._normalize_array_index(
-                            start, edgedb._length(val))
-                    )
-            END
+            edgedb._substr(
+                val,
+                edgedb._normalize_array_index(
+                    start, edgedb._length(val)),
+                edgedb._normalize_array_index(
+                    stop, edgedb._length(val)) -
+                edgedb._normalize_array_index(
+                    start, edgedb._length(val))
+            )
     '''
 
     def __init__(self) -> None:
         super().__init__(
             name=('edgedb', '_str_slice'),
             args=[
-                ('val', ('anyelement',)),
```

Comment:
I would leave these arguments one per line. It all fits on one, but especially with this sort of nested data I think the line breaks help.

Feel free to add trailing commas, which should calm down your code formatter?